File: src/main/java/com/max/Ticket.java
package com.max;

public class Ticket {
    int origin;
    int destination;
    int price;

    public Ticket(int origin, int destination, int price) {
        this.origin = origin;
        this.destination = destination;
        this.price = price;
    }

    public Ticket() {
    }

    public int getOrigin() {
        return origin;
    }

    public void setOrigin(int origin) {
        this.origin = origin;
    }

    public int getDestination() {
        return destination;
    }

    public void setDestination(int destination) {
        this.destination = destination;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
}

File: src/main/kotlin/com/max/Ticket.kt
package com.max

import java.lang.Math.random
import kotlin.random.Random

data class Ticket(val origin:Int, val destination:Int, val price:Int)

fun main(args: Array<String>) {
    var ticket = Ticket(20,51,420)
    val s = "abcde"
    println(s.validate())
    println("Ticket Done")
    println((1..10).random())
}

fun String.validate() : Boolean {
    return this.length >= 6
}

fun IntRange.random(): Int {
    val r = Random(2).nextInt(endInclusive-start)+start
    return r
}
